### DISCUSSION TITLE
How to use Cloud Sync

### DISCUSSION THREAD

**Discussion by 1RonRock** [sienori/Tab-Session-Manager#570] [Q&A] [answered]
I have had the Cloud Sync checked in TSM since it was first added. The sync seems to work if I go into TSM settings and click the cloud sync icon at the top of the page. Not sure if the cloud sync is automatic or needs to be done manually.Where in my Google drive is the cloud sync folder? I can not find it or the path that it should be at.

**Comment by sienori** [comment, marked as answer]
You can enable cloud sync from the settings page.
![sync settings](https://user-images.githubusercontent.com/29514424/78505793-c027ba80-77b0-11ea-80ed-de25f8e67552.png)

Cloud sync can be done manually by clicking on the sync icon.
![sync icon](https://user-images.githubusercontent.com/29514424/78505794-c322ab00-77b0-11ea-8347-2744ea0f6494.png)

It will be saved as hidden app data in Google Drive.
You can check the capacity from the Drive `Settings`>`Manage Apps`.
![settings](https://user-images.githubusercontent.com/29514424/101611127-76345a00-3a4c-11eb-8545-4440b8fc7f24.png)
![manage apps](https://user-images.githubusercontent.com/29514424/101611029-53a24100-3a4c-11eb-8b4c-1bab166e64e0.png)

**Comment by rediffusion** [comment]
It doesn’t work for me because I still see the `exclamation mark`.
